_posts/2019-10-29-Subnero-at-iOcean-Sanya.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: Subnero at iOceans 2019, Sanya
+banner : images/banner-pulse-standard.jpg
+date:  2019-10-29
+categories: news
+thumbnail: images/pulse-thumbnail-smartsea-team.jpg
+---
+
+Subnero was delighted to be with [Wuxi Smartsea Technology](http://www.smartseatech.com/en/index.aspx){:target="_blank"} at iOceans 19.
+
+The event took place from the 25th to the 27th of October, in Sanya, China.
+
+Details of the exhibition can be found [here](http://www.cso.org.cn/ggxx/2019/0926/2816.html){:target="_blank"}.
```

Comment:
Can we remove this sentence and move this hyperlink onto "The event" in the previous line. Or some other part of the post which is relevant to the link?

Reply:
Done.